src/utils/__tests__/url-test.js

```diff
@@ -514,3 +515,40 @@ describe('autocompleteUrl', () => {
     expect(result).toEqual('https://subdomain.mydomain.org');
   });
 });
+
+describe('extract actual stream name from streamId appended version', () => {
```

Comment:
We use the function name in the `describe` call, so `extractStreamName` in this case.

Reply:
Uh.. Fixed. ☑️